magda-web-client/package.json

```diff
@@ -93,6 +97,8 @@
     "echarts-for-react": "^2.0.14",
     "email-validator": "^2.0.3",
     "eslint": "^5.14.1",
+    "eslint-plugin-flowtype": "2.50.1",
```

Comment:
Why do we need flowtype?

Reply:
It's required by `es-lint`. the `eslint` can't find it anymore due to the `no hoist` setting to `react-scripts`
See [logs](https://gitlab.com/magda-data/magda/-/jobs/475878013)